### PR TITLE
Tasks Mark +layout

### PR DESCRIPTION
**Pull request by markru37**

### CODE REVIEW

File: CourseApp/Program.cs [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
-        }
-
-        public static void OutputTask(List<double> listTask)
-        {
-            foreach (var y in listTask)
+            string[] phraseArray = { "Грустно:(", "Как же печально:(", "Жалко животинку:(" };
+            Pig pigOne = new Pig();
+            Pig pigTwo = new Pig("свинка Пеппа", 9, 99);
+            Boar boarOne = new Boar("кабанчик Кабанистый", 1, 5, 7);
+            Boar boarTwo = new Boar("кабанчик Крутечок", 3, 2, 8);
```
Comment (jskonst) [inline CourseApp/Program.cs:14]: var pigOne ...

Comment (jskonst) [reply CourseApp/Program.cs:14]: Animal pigTwo = new Pig()

File: CourseApp/Pig.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            Lard = 0;
+            return $"{Name} зарезана\nПолучено {lard} сала\n";
+        }
+
+        public override string MakePhrase(string[] phraseArray)
+        {
+            int index = random.Next(0, 3);
+            string phrase = phraseArray[index];
+            return $"{phrase}";
+        }
```
Comment (jskonst) [inline CourseApp/Pig.cs:47]: What if i will pass you empty array?

File: CourseApp/Pig.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+        public override string Died()
+        {
+            int lard = Lard;
+            Lard = 0;
+            return $"{Name} зарезана\nПолучено {lard} сала\n";
+        }
+
+        public override string MakePhrase(string[] phraseArray)
```
Comment (jskonst) [inline CourseApp/Pig.cs:42]: string[] phraseArray -> this.string[] phraseArray = ["phrases1", "phrases1", "phrases1"]

File: CourseApp/Pig.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            Lard += lard;
+        }
+
+        public override string Died()
+        {
+            int lard = Lard;
+            Lard = 0;
+            return $"{Name} зарезана\nПолучено {lard} сала\n";
+        }
```
Comment (jskonst) [inline CourseApp/Pig.cs:40]: can we get some Lard from dead pig?

File: CourseApp/Pig.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+        public Pig(string name, int weight, int age, int lard)
+        : base(name, weight, age, lard)
+        {
+        }
+
+        public void AddLard(int lard)
+        {
+            Lard += lard;
+        }
```
Comment (jskonst) [inline CourseApp/Pig.cs:33]: Do we need this one?

File: CourseApp/Boar.cs [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+        public Boar(string name, int weight, int age, int lard)
+        : base(name, weight, age, lard)
+        {
+        }
+
+        public override string Died()
+        {
+            int lard = Lard;
+            Lard = 0;
```
Comment (jskonst) [inline CourseApp/Boar.cs:33]: you could implement this ion base class

File: CourseApp/Animals.cs [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                if (value < 0)
+                {
+                    lard = 0;
+                }
+                else
+                {
+                    lard = value;
+                }
+            }
+        }
```
Comment (jskonst) [inline CourseApp/Animals.cs:99]: All animals have Lard?

**Comment by markru37** [comment]
https://markru37.github.io/aboutmyself/

**Pull request closed without merging**